engine/platform/win32/con_win.c

```diff
@@ -29,6 +29,9 @@ WIN32 CONSOLE
 
 typedef struct
 {
+	qboolean    is_attached;
+
+	char        old_title[512];
```

Comment:
You can straight away use types like string, which is usually long enough to store generic strings.

Reply:
Fix in [a70fa30](https://github.com/FWGS/xash3d-fwgs/pull/848/commits/a70fa304ee1e0a1bae1de4a2fb26346578246f56)